build.gradle

```diff
@@ -99,8 +99,8 @@ subprojects { subproject ->
 		log4j2Version = '2.7'
 		logbackVersion = '1.1.7'
 		mockitoVersion = '1.10.19'
-		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '3.6.5'
-		rabbitmqHttpClientVersion = '1.0.0.RELEASE'
+		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '4.0.0'
```

Comment:
I guess the main reason of this upgrade is a robust auto-recovery function which is enabled by default now.
Seems for me there was even some community request to provide this upgrade earlier than Spring AMPQ 2.0...